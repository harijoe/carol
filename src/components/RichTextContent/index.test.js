import React from 'react'
import { shallow } from 'enzyme'

import RichTextContent from '.'

it('renders the RichTextContent component', () => {
  expect(shallow(<RichTextContent content={'Some text with a <a href="https://google.com">link</a>.'} />)).toMatchSnapshot()
})
