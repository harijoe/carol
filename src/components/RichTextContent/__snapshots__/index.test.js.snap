// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the RichTextContent component 1`] = `
<StyledParagraph
  dangerouslySetInnerHTML={
    Object {
      "__html": "Some text with a <a href=\\"https://google.com\\">link</a>.",
    }
  }
/>
`;
